_pages/Long-term-support/index.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: default
+permalink: /Long-term-support/
+---
+
+## Long term engineering support
+
+## Proposal support
```

Comment:
The page is called long-term-support, but that is actually only one section of the page. You have also Proposal support, Data support etc. That can be confusing. Also the Proposal Support section is a repetition of what is already on the landing page. I would either only talk about funding on this page (and rename the page funding), or restructure more to make it consistent.

Reply:
Changed the header to "research and grant proposals" and removed the duplicate information. Hope this is more consistent